From 81232cc71c888de71202e19dbe38786ee161e219 Mon Sep 17 00:00:00 2001
From: Kondapally Kalyan <kalyan.kondapally@intel.com>
Date: Tue, 17 Jun 2014 19:13:41 +0300
Subject: [PATCH 1/5] Temporary patch to fix linking issue.

---
 ui/views/views.gyp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ui/views/views.gyp b/ui/views/views.gyp
index 101245a..9e1699b 100644
--- a/ui/views/views.gyp
+++ b/ui/views/views.gyp
@@ -389,6 +389,8 @@
       'widget/desktop_aura/desktop_capture_client.cc',
       'widget/desktop_aura/desktop_capture_client.h',
       'widget/desktop_aura/desktop_cursor_loader_updater.h',
+      'widget/desktop_aura/desktop_cursor_loader_updater_auralinux.cc',
+      'widget/desktop_aura/desktop_cursor_loader_updater_auralinux.h',
       'widget/desktop_aura/desktop_dispatcher_client.cc',
       'widget/desktop_aura/desktop_dispatcher_client.h',
       'widget/desktop_aura/desktop_drop_target_win.cc',
@@ -407,8 +409,6 @@
       'widget/desktop_aura/desktop_screen_position_client.h',
     ],
     'views_desktop_aura_x11_sources': [
-      'widget/desktop_aura/desktop_cursor_loader_updater_auralinux.cc',
-      'widget/desktop_aura/desktop_cursor_loader_updater_auralinux.h',
       'widget/desktop_aura/desktop_drag_drop_client_aurax11.cc',
       'widget/desktop_aura/desktop_drag_drop_client_aurax11.h',
       'widget/desktop_aura/desktop_window_tree_host_x11.cc',
-- 
1.7.9.5

