From 84690ea4da5dcf42371d9b932cf5dc581dacdecd Mon Sep 17 00:00:00 2001
From: Joone Hur <joone.hur@intel.com>
Date: Tue, 1 Nov 2016 22:52:56 -0700
Subject: [PATCH 14/18] don't include exo

---
 components/components.gyp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/components.gyp b/components/components.gyp
index bbf5779..1d0ab8a 100644
--- a/components/components.gyp
+++ b/components/components.gyp
@@ -212,7 +212,7 @@
         'rlz.gypi',
       ],
     }],
-    ['use_ash==1', {
+    ['use_ash==1 and enable_wayland_server==1', {
       'includes': [
         'exo.gypi',
       ],
-- 
2.7.4

