From 58641a34e5a950cf37af64ae8bf6ce416a19e73b Mon Sep 17 00:00:00 2001
From: Joone Hur <joone.hur@intel.com>
Date: Fri, 4 Nov 2016 16:13:21 -0700
Subject: [PATCH 16/16] remove views_desktop_aura_ozone_sources

---
 ui/views/views.gyp | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/ui/views/views.gyp b/ui/views/views.gyp
index b5b122e..f7032c6 100644
--- a/ui/views/views.gyp
+++ b/ui/views/views.gyp
@@ -486,12 +486,6 @@
       'widget/desktop_aura/desktop_window_tree_host_win.cc',
       'widget/desktop_aura/desktop_window_tree_host_win.h',
     ],
-    'views_desktop_aura_ozone_sources': [
-      'widget/desktop_aura/desktop_factory_ozone.cc',
-      'widget/desktop_aura/desktop_factory_ozone.h',
-      'widget/desktop_aura/desktop_screen_ozone.cc',
-      'widget/desktop_aura/desktop_window_tree_host_ozone.cc',
-    ],
     'views_test_support_sources': [
       'animation/test/flood_fill_ink_drop_ripple_test_api.cc',
       'animation/test/flood_fill_ink_drop_ripple_test_api.h',
@@ -816,9 +810,6 @@
             ['OS == "win"', {
               'sources': [ '<@(views_desktop_aura_win_sources)' ],
             }],
-            ['use_ozone==1', {
-              'sources': [ '<@(views_desktop_aura_ozone_sources)' ],
-            }],
           ],
         }],
         ['OS=="mac"', {
-- 
2.7.4

